src/storage/engine/raftkv.rs

```diff
@@ -286,6 +286,7 @@ impl<S: RaftStoreRouter> Engine for RaftKv<S> {
         mut modifies: Vec<Modify>,
         cb: Callback<()>,
     ) -> engine::Result<()> {
+        fail_point!("async_write");
```

Comment:
now failpoint removes the mod name, so to avoid duplicated name, I think we should use a detailed name like raftkv_async_write. 

/cc @overvenus